src/Microsoft.DotNet.Interactive.Http/Model/HttpNamedRequest.cs

```diff
@@ -171,19 +180,24 @@ internal HttpNamedRequest(HttpRequestNode httpRequestNode, HttpResponse response
                     var xmlDoc = new XmlDocument();
                     xmlDoc.LoadXml(Response.Content.Raw);
 
-
-                    //Remove the leading slash
-                    var xmlNodes = xmlDoc.SelectNodes(path[3].Substring(1));
-
-                    if (xmlNodes is not null && xmlNodes.Count == 1)
+                    try
                     {
-                        return node.CreateBindingSuccess(xmlNodes.Item(0)?.Value);
+                        //Remove the leading slash
+                        var xmlNodes = xmlDoc.SelectNodes(path[3].Substring(1));
+
+                        if (xmlNodes is not null && xmlNodes.Count == 1)
```

Comment:
```suggestion
                        if (xmlNodes is { Count : 1} )
```

This suggestion effectively combines the null check and the property check.